libs/solid/skeleton/src/lib/services/feedback.service.ts

```diff
@@ -24,25 +23,28 @@ export function feedbackServiceFactory(
 }
 
 export class FeedbackService {
-  /**
-   * use this variable to save the value of the component's url 
-   * 1. if the user open it from the toolbar => the url will be "menu"
-   * 2. if the user open it from the sidebar(Menu) => the url will be the current's url 
-   */
-  private currentUrl : string = ""; 
-  
   constructor(
     private _http: HttpClient,
     private _dialog: MatDialog,
     private _config: SolidCoreConfig
   ) {}
 
-  public showDialog() {
-    this._dialog.open(FeedbackComponent, {
+  public showDialog(location?: any) {
+    const dialogConfig = new MatDialogConfig();
+    let title = 'Kontakt und Feedback';
+    let subject = 'Feedback';
+    if (location != undefined) {
+      title = 'Fehler melden';
+      subject = 'Fehler melden';
+    }
+    dialogConfig.data = {
```

Comment:
Houston, we have a problem.
Ich habe das jetzt nochmal genau angeschaut - mit dieser Technik funktioniert die `submitFeedback()`-Funktion nicht mehr.

`width` und `maxWidth` sind Eigenschaften von `dialogConfig` (siehe [AngularMaterial Specs](https://material.angular.io/components/dialog/api#MatDialogConfig)). Du machst sie jetzt aber zum Teil von `dialogConfig.data`. Dann werden sie gar nicht ausgewertet.

Reply:
Es müsste sein
```
dialogConfig.width = '80%'
dialogConfig.maxWidth = '800px'
```